client/src/theme/palette.js

```diff
@@ -0,0 +1,44 @@
+export const applyAlpha = (alpha, color) => color + alpha;
```

Comment:
@Spiteless can you help me understand this `applyAlpha` ?

it is exported here but doesn't seem to be imported anywhere

Reply:
lol nevermind just read your explanation in PR creation comment